tests/all/integration/api/group/test_group_misc.py

```diff
@@ -0,0 +1,140 @@
+import json
+
+from flask_jwt_extended import create_access_token
+
+from app.api.helpers.db import get_or_create
+from app.models.role import Role
+from app.models.users_events_role import UsersEventsRoles
+from tests.factories.event import EventFactoryBasic
+from tests.factories.group import GroupFactory
+from tests.factories.user import UserFactory
+
+
+def test_group_post_access_allow(db, client):
+    new_user = UserFactory(is_admin=False, is_verified=True)
+    event = EventFactoryBasic()
+    role, _ = get_or_create(Role, name='coorganizer', title_name='Co-organizer')
+    UsersEventsRoles(user=new_user, event=event, role=role)
+
+    db.session.commit()
+    data = json.dumps(
+        {
+            "data": {
+                "type": "group",
+                "relationships": {
+                    "user": {"data": {"id": str(new_user.id), "type": "user"}},
+                    "events": {"data": [{"id": str(event.id), "type": "event"}]},
+                },
+                "attributes": {"name": "eventgp2"},
+            }
+        }
+    )
+
+    response = client.post(
+        f'/v1/groups',
+        content_type='application/vnd.api+json',
+        headers={'Authorization': 'JWT ' + create_access_token(new_user.id, fresh=True)},
```

Comment:
No need of creating a user and jwt, just add user and jwt as paramters in the test function, they'll be injected automatically

See other tests for reference

Reply:
@iamareebjamal sir I had done that initially but by default user is_verified is false so I also tried setting user.is_verified but that was throwing error so I used new user is there any way to bypass this?